lib/omniauth_keycloak/configuration.rb

```diff
@@ -49,7 +49,9 @@ def discovery_url
   end # #discovery_url
 
   def discovery_object
-    HTTParty.get(discovery_url) || {}
+    unless Rails.env.test?
+      HTTParty.get(discovery_url) || {}
+    end
```

Comment:
```suggestion
    return if Rails.env.test?
    
    HTTParty.get(discovery_url) || {}
```

preferably use a guard clause :) and why do you need to check for test env anyway?

Reply:
Why dont you stub the endpoint in your tests then? 🤔  

```ruby
allow(HTTParty).to receive(:get).and_return({...})
```

Reply:
The issue occurs during the init step not during the rspec tests, so we don't get to that point to stub it.
Anyway I remove the line because I totally forgot that MF and I agree to add env variable to our spec helper to get this running.